modules/dists/BlockDist.chpl

```diff
@@ -1273,20 +1294,29 @@ proc BlockDom.dsiSupportsPrivatization() param return true;
 record BlockDomPrvData {
   var distpid;
   var dims;
+  var locdoms;  //todo rvf its elements along with the rest of the record
```

Comment:
This PR does not require this. I am leaving this as future work.

Aside: adding this field to `BlockDomPrvData` saves 2 remote gets per privatizing locale. No big deal, but I thought I'd add it anyway.

Reply:
I meant to ask this question about privatization rather than RVF, but my brain failed me. If @mppf is OK with this being part of the PR then it's fine with me.

Reply:
I wasn't sure but I suspected that it might be related to the comm counts changes. Other elements of the PR have the potential to change comms (maybe increase them?) so it seems better (at this point) to keep it all in one PR.

@vasslitvinov - I agree with @benharsh that it would be ideal to do this in a different PR, but I do not want to insist on it right now.